### PR TITLE
[crewplanning] making crewplanning buildable

### PR DESCRIPTION
**Pull request by guicho271828**

### CODE REVIEW

File: crewplanning/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+all:
+	ocamlbuild main.native
+
+clean:
+	ocamlbuild -clean
```
Comment (jendrikseipp) [inline crewplanning/Makefile:6]: Could you add a "test" target here that executes the generator, please?

**Comment by guicho271828** [comment]
I don't think this build will pass because the CI container probably lacks Ocaml compiler

**Comment by jendrikseipp** [comment]
Can you add one to the GitHub Action?

**Comment by guicho271828** [comment]
done

**Comment by jendrikseipp** [comment]
Nice, thanks!

**Pull request merged**